satellite/console/service.go

```diff
@@ -55,37 +56,36 @@ const (
 
 // Service is handling accounts related logic
 type Service struct {
-	Signer
+	log *zap.Logger
 
-	store DB
-	log   *zap.Logger
+	Signer
```

Comment:
Can we split external from internal vars into blocks?
Dont think its necessary, but makes the code clearer imo